tests/test.py

```diff
@@ -107,13 +120,22 @@ def test_search(self):
                          int(result['search_metadata']['maximumRecords']))
         self.assertEqual(int(result['search_metadata']['maximumRecords']), 18)
 
+        result = self.loklak.search('FOSSASIA', since='2000-01-01', until='2017-12-31_12:24', from_user='fossasia', count=20)
+        self.assertTrue('statuses' in result)
+
+        result = self.loklak.search()
+        self.assertEqual(result, '{"error": "No Query string has been given to query for an account"}')
+
     def test_aggregations(self):
         """Test aggregations."""
         result = self.loklak.aggregations('fossasia', '2017-01-10',
                                           '2018-01-10', 10)
         data = result.json()
         self.assertEqual(result.status_code, 200)
         self.assertTrue('aggregations' in data)
+
+        result = self.loklak.aggregations()
+        self.assertEqual(result, '{"error": "No Query string has been given to run query for aggregations"}')
     #     self.assertTrue('hashtags' in data['aggregations'])
```

Comment:
Have you found solution to include fields? Please see this https://github.com/loklak/loklak_python_api/pull/75#issuecomment-358653952 for more info

Reply:
Not yet, but I will check it out soon.